src/diffusers/pipelines/stable_diffusion/convert_from_ckpt.py

```diff
@@ -644,6 +669,10 @@ def convert_ldm_clip_checkpoint(checkpoint):
         if key.startswith("cond_stage_model.transformer"):
             text_model_dict[key[len("cond_stage_model.transformer.") :]] = checkpoint[key]
 
+    # if missing text encoder keys, checkpoint may be pruned so just use pretrained weights
+    if text_model_dict:
+        text_model.load_state_dict(text_model_dict)
+
     text_model.load_state_dict(text_model_dict)
```

Comment:
I'm not sure I follow, it looks like this is calling load_state_dict twice regardless?